src/services/backend.service.ts

```diff
@@ -1,9 +1,8 @@
 import { Injectable } from '@angular/core';
 import { Http, Headers } from '@angular/http';
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
 import { map, catchError } from 'rxjs/operators';
-
-import { User } from '../datatypes/datatypes';
+import { User, Game } from '../datatypes/datatypes';
 
 const createHeaders = token => new Headers({'Authorization': 'Bearer ' + token});
```

Comment:
We can delete this header 🤔

Reply:
Since we plan on replacing the "**any**" datatypes **any**ways, no

Reply:
Oh I mean the unused constant createHeaders
Lol this pun.